src/core/storage/Position.h

```diff
@@ -1,12 +1,19 @@
 #pragma once
 
+#include <nop/serializer.h>
+#include <nop/structure.h>
+
+#include <fmt/format.h>
+#include <fmt/core.h>
+
 #include <functional>
 #include <string>
 
 namespace internal::storage {
 
 class Position {
 	friend std::hash<Position>;
+
 public:
 	Position() = default;
 	Position(const Position &) = default;
```

Comment:
`explicit Position(long pos) : m_pos{pos}, m_isset{true} {}`
`explicit operator long() const { return m_pos; }`
in 26 and 27 rows

Reply:
No. I am intentionally making this implictly convertible to `long`.